pkg/build.sh

```
@@ -8,7 +8,7 @@ TARGET="release"
 MODELFILE_DIR="modelfiles"
 SERVER_DIR="server"
 BINARY_NAME="tiles"
-
+MODELS_DIR="models"
```

Shellcheck (0.11.0)

[warning] MODELS_DIR appears unused. Verify use (or export if used externally).

(SC2034)